File: GoodOfflineGames/GOG.Models/AccountProduct.cs
using System.Runtime.Serialization;

namespace GOG.Models
{
    [DataContract]
    public class AccountProduct: ProductCore
    {
        [DataMember(Name = "isGalaxyCompatible")]
        public bool IsGalaxyCompatible { get; set; }
        [DataMember(Name = "tags")]
        public Tag[] Tags { get; set; }
        [DataMember(Name = "availability")]
        public Availability Availability { get; set; }
        [DataMember(Name = "image")]
        public string Image { get; set; }
        [DataMember(Name = "url")]
        public string Url { get; set; }
        [DataMember(Name = "worksOn")]
        public WorksOn WorksOn { get; set; }
        [DataMember(Name = "category")]
        public string Category { get; set; }
        [DataMember(Name = "rating")]
        public int Rating { get; set; }
        [DataMember(Name = "isComingSoon")]
        public bool IsComingSoon { get; set; }
        [DataMember(Name = "isMovie")]
        public bool IsMovie { get; set; }
        [DataMember(Name = "isGame")]
        public bool IsGame { get; set; }
        [DataMember(Name = "slug")]
        public string Slug { get; set; }
        [DataMember(Name = "updates")]
        public int Updates { get; set; }
        [DataMember(Name = "isNew")]
        public bool IsNew { get; set; }
        [DataMember(Name = "dlcCount")]
        public int DLCCount { get; set; }
        [DataMember(Name = "releaseDate")]
        public TimezoneDate ReleaseDate { get; set; }
        [DataMember(Name = "isBaseProductMissing")]
        public bool IsBaseProductMissing { get; set; }
        [DataMember(Name = "isHidingDisabled")]
        public bool IsHidingDisabled { get; set; }
        [DataMember(Name = "isInDevelopment")]
        public bool IsInDevelopment { get; set; }
        [DataMember(Name = "isHidden")]
        public bool IsHidden { get; set; }
    }
}

File: internal/gog/accountProducts/accountProduct.go
package accountProducts

const (
	MediaTypeGame  = 1
	MediaTypeMovie = 2
)

type AccountProduct struct {
	IsGalaxyCompatible bool          `json:"isGalaxyCompatible"`
	Tags               []interface{} `json:"tags"`
	ID                 int           `json:"id"`
	Availability       struct {
		IsAvailable          bool `json:"isAvailable"`
		IsAvailableInAccount bool `json:"isAvailableInAccount"`
	} `json:"availability"`
	Title   string `json:"title"`
	Image   string `json:"image"`
	URL     string `json:"url"`
	WorksOn struct {
		Windows bool `json:"Windows"`
		Mac     bool `json:"Mac"`
		Linux   bool `json:"Linux"`
	} `json:"worksOn"`
	Category     string `json:"category"`
	Rating       int    `json:"rating"`
	IsComingSoon bool   `json:"isComingSoon"`
	IsMovie      bool   `json:"isMovie"`
	IsGame       bool   `json:"isGame"`
	Slug         string `json:"slug"`
	Updates      int    `json:"updates"`
	IsNew        bool   `json:"isNew"`
	DlcCount     int    `json:"dlcCount"`
	ReleaseDate  struct {
		Date         string `json:"date"`
		TimezoneType int    `json:"timezone_type"`
		Timezone     string `json:"timezone"`
	} `json:"releaseDate"`
	IsBaseProductMissing bool `json:"isBaseProductMissing"`
	IsHidingDisabled     bool `json:"isHidingDisabled"`
	IsInDevelopment      bool `json:"isInDevelopment"`
	IsHidden             bool `json:"isHidden"`
}
